DigitalLearningSolutions.Web/Views/TrackingSystem/Shared/_DelegatesSideNavMenu.cshtml

```diff
@@ -0,0 +1,30 @@
+﻿@using Microsoft.Extensions.Configuration
+@inject IConfiguration Configuration
+
+@{
+  var title = ViewData["Title"].ToString();

```

Comment:
Not 100% sure about making this so decoupled with the ViewData title (which should be a purely UI value) - maybe we can pass in a value for the "selectedPage" which is from a list enums?